ynput/OpenPype
# AfterEffects: added validator for missing files in FootageItems

## Changelog Description
Published composition in AE could contain multiple FootageItems as a layers. If FootageItem contains imported file and it doesn't exist, render triggered by Publish process will silently fail and no output is generated. This could cause failure later in the process with unclear reason. (In `ExtractReview`).
This PR adds validation to protect from this.

## Additional info
Currently it doesn't check for missing frame in image sequences as this doesn't cause crash in rendering and it would be quite difficult to check if FootageItem should contain image sequence as AE api doesn't provide that info.

!!Requires reinstall of extension !!

## Testing notes:
1. Load image to AE (via Loader or just regular AE Import (right click in list of items in AE). It must be single file!
2. remove or rename that file on the disk
3. validator should trigger

![AnyDeskMSI_kVZ7Ab1etc](https://github.com/ynput/OpenPype/assets/4457962/620f1420-9b77-4c42-9cf6-a9ffa2ab17de)
